service/templates/projects.html

```diff
@@ -472,14 +472,45 @@ <h2> Settings for [[displayName]]</h2>
                 <div class="col-lg-2">
                   <tag-input klass="form-control" items="github_alias.tags" placeholder="tags (comma-delimited)" />
                 </div>
-                <div class="col-lg-4">
+                <div class="col-lg-2">
                   <button class="plus-button btn btn-primary" ng-disabled="!validPatchDefinition(github_alias)" type="button" ng-click="addGithubAlias()">
                     <i class="fa fa-plus"></i>
                   </button>
                 <label class="distro-error">[[invalidGitHubPatchDefinitionMessage]]</label>
                 </div>
+            </div> 
+
+            <!--  Github Trigger Aliases  -->
+            <div id="github-trigger-alias">
+              <div class="h4">Add Github Trigger Alias </div>
+              <div class="muted small">Add previously defined aliases that add tasks from other projects to patches</div>
```

Comment:
Maybe "Add previously defined _patch trigger_ aliases"? This would disambiguate them from patch aliases